Detectors/Base/src/MaterialManager.cxx

```diff
@@ -91,21 +93,66 @@ const std::unordered_map<ECut, const char*> MaterialManager::mCutIDToName = {
   {ECut::kPPCUTM, "PPCUTM"},
   {ECut::kTOFMAX, "TOFMAX"}};
 
+// Constructing a map between module names and local material density values
+void MaterialManager::createDensityMap()
+{
+  auto& globalDensityFactor = o2::conf::SimMaterialParams::Instance().globalDensityFactor;
+  if (globalDensityFactor < 0) {
+    LOG(fatal) << "Negative value "
+               << globalDensityFactor
+               << " found for global material density!\n";
+  }
+  std::vector<std::string> allModuleNames = {
+    "ABSO", "CAVE", "COMP", "DIPO", "FRAME", "HALL", "MAG", "PIPE",
+    "SHIL", "CPV", "EMC", "FDD", "FT0", "FV0", "HMP", "ITS",
+    "MCH", "MFT", "MID", "PHS", "TOF", "TPC", "TRD", "ZDC",
+    "ALPIDE", "IT3", "TRK", "FT3", "A3IP"};
```

Comment:
do we really need this static list of names? I think we better should avoid this because otherwise we might run into troubles when someone adds more detectors.
It should be enough to create the map dynamically from the given string. One can then create another helper and return `globalDensityFactor` whenever something is not in the map, or in the other case the content of the map.